backend/program/scrapers/jackett.py

```diff
@@ -86,59 +103,204 @@ def scrape(self, item: MediaItem) -> MediaItem:
 
     def api_scrape(self, item: MediaItem) -> tuple[Dict[str, Torrent], int]:
         """Wrapper for `Jackett` scrape method"""
-        with self.minute_limiter:
-            query = ""
+
+        indexers = self._get_indexers()

```

Comment:
what happens if this is empty?

Reply:
If it's empty, there will just be no results. But maybe we should log a warning?

Reply:
we should validate if the user has any indexers enabled before initializing the jackett scraper.. if no indexers are found and the user has it enabled, we should error log, and not initialize